.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
```

Comment:
As far as the master branch is concerned, it might be better to just run it periodically instead of using push events.